File: market-application/src/main/java/com/example/market/application/port/in/MarkPayoutCompletedUseCase.java
package com.example.market.application.port.in;

import com.example.market.domain.settlement.PayoutId;

/**
 * 은행 송금 완료 콜백 → Payout.complete.
 */
public interface MarkPayoutCompletedUseCase {
    void markCompleted(PayoutId payoutId);
}

File: market-application/src/main/kotlin/com/example/market/application/port/in/MarkPayoutCompletedUseCase.kt
package com.example.market.application.port.`in`

import com.example.market.domain.settlement.PayoutId

/**
 * 은행 송금 완료 콜백 → Payout.complete.
 */
interface MarkPayoutCompletedUseCase {
    fun markCompleted(payoutId: PayoutId)
}
